packages/react-native/src/theme/types.ts

```diff
@@ -44,6 +45,7 @@ export interface ComponentStyles {
   label: LabelStyles;
   radio: RadioStyles;
   radiogroup: RadioGroupStyles;
+  textfield: TextFieldStyles;
```

Comment:
Why are we using lowercase instead of camelCase for `textfield` (and `radiogroup`)?

Reply:
I was just following [this pattern](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/components/index.ts#L93).